moped-editor/src/utils/localStorageHelpers.js

```diff
@@ -0,0 +1,77 @@
+import { useState, useEffect } from "react";
+
+/**
+ * Get the user hidden column settings from local storage, clear them of
+ * outdated columns, and supplement with remaining default columns
+ * @param {Object} defaultColumnSettings - the baseline column visibility settings, where each entry
+ * is a column name with a boolean value indicating if the column should be hidden
+ * * @param {String} storageKey - the key used to get and set data from local storage
+ * @returns {Object} hidden column settings from local storage
+ */
+const getPreviousHiddenColumns = (defaultColumnSettings, storageKey) => {
```

Comment:
the only changes here is that these functions now take `defaultColumnSettings` and the `storageKey` args